tests/pagination/tests.py

```diff
@@ -116,12 +117,18 @@ def test_invalid_page_number(self):
         Invalid page numbers result in the correct exception being raised.
         """
         paginator = Paginator([1, 2, 3], 2)
-        with self.assertRaises(InvalidPage):
+        with self.assertRaises(InvalidPage) as e:
             paginator.page(3)
-        with self.assertRaises(PageNotAnInteger):
+        self.assertEqual(e.exception.message, _('That page contains no results'))
+        with self.assertRaises(PageNotAnInteger) as e:
             paginator.validate_number(None)
-        with self.assertRaises(PageNotAnInteger):
+        self.assertEqual(e.exception.message, _('That page number is not an integer'))
+        with self.assertRaises(PageNotAnInteger) as e:
             paginator.validate_number('x')
+        self.assertEqual(e.exception.message, _('That page number is not an integer'))
+        with self.assertRaises(EmptyPage) as e:
+            paginator.validate_number(0)
+        self.assertEqual(e.exception.message, _('That page number is less than 1'))
```

Comment:
@claudep You are right, It's really did not test anything. I removed this changes. Regarding translations, I'll add strings (russian) after you accept PR.